src/components/tracked-controls.js

```diff
@@ -145,12 +145,12 @@ module.exports.Component = registerComponent('tracked-controls', {
     // Compose pose from Gamepad.
     pose = controller.pose;
     // If no orientation, use camera.
-    if (pose.orientation) {
+    if (pose && pose.orientation) {
```

Comment:
I don't think can have position w/o orientation, pretty sure removing 'hasOrientation' from WebVR spec for that reason